prql-compiler/src/semantic/declarations.rs

```diff
@@ -80,7 +80,7 @@ impl Debug for Declarations {
 impl Display for Declaration {
     fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
         match self {
-            Declaration::Expression(node) => write!(f, "{}", node.item),
+            Declaration::Expression(node) => write!(f, "({})", node.item),
```

Comment:
For instance, when generating PRQL for a `table`, the parentheses were missing: 
```
table top_employees
from employees
sort [rating]
take 20
```

Thanks to this code, we now get: 
```
table top_employees (
  from employees
  sort [rating]
  take 20
)
```